tests/kani/Intrinsics/Atomic/Unstable/AtomicMin/main.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Check that `atomic_min` and other variants (unstable version) return the
+// expected result.
+
+#![feature(core_intrinsics)]
+use std::intrinsics::{
+    atomic_min, atomic_min_acq, atomic_min_acqrel, atomic_min_rel, atomic_min_relaxed,
+};
+
+#[kani::proof]
+fn main() {
+    let mut a1 = 1 as u8;
```

Comment:
Is there a reason we only test for `u8`? Could we make this take a `<T>` and then test multiple bitwidths?

Reply:
Yes, that's pending in #25. But I'd like to do it once all atomic tests are in.